content/events/hacktoberfest.adoc

```diff
@@ -14,56 +14,52 @@ opengraph:
+** Your pull or merge request must not be labeled as invalid to be counted.
+** For your pull or merge requests to be accepted, your pull or merge requests must be merged, or have the `hacktoberfest-accepted` label, or have an overall approving review on either GitHub or GitLab.
 
 == Where to contribute?
 
 The Jenkins project is spread across several organizations on GitHub (`link:https://github.com/jenkinsci/[jenkinsci]`, `link:https://github.com/jenkins-infra/[jenkins-infra]`, `link:https://github.com/jenkins-zh/[jenkins-zh]`).
 You are welcome to contribute to **any** repository in **any** of those organizations, or any other Jenkins-related repository on GitHub.
 Repositories may have different contribution guidelines, review and merge policies.
-If you adopt Jenkins in your open-source projects (e.g. Jenkins Pipeline or Configuration as Code),
-it counts as well!
+If you adopt Jenkins in your open-source projects (e.g. Jenkins Pipeline or Configuration as Code), it counts as well!
 Note that not all pull requests will automatically count towards Hacktoberfest.
 
-* link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+org%3Ajenkins-zh+topic%3Ahacktoberfest[List of repositories marked for Hacktoberfest] -
-  you can just submit pull requests, no extra steps needed.
-* for other repositories, we will need to get the repo marked for Hacktoberfest so that your pull request counts towards the goal.
+* link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+org%3Ajenkins-zh+topic%3Ahacktoberfest[List of repositories marked for Hacktoberfest] - you can just submit pull requests, no extra steps needed.
+* For other repositories, we will need to get the repo marked with the topic "Hacktoberfest" for your pull request to count towards the goal.
   Please follow link:/events/hacktoberfest/faq/#how-do-i-mark-my-pull-requests[FAQ: Marking Pull requests].
 
-=== Issue queries
-
-We have marked some issues in Jenkins Jira and GitHub issues which can be handled by contributors during Hacktoberfest:
-
-* link:https://issues.jenkins.io/issues/?jql=labels%20%3D%20hacktoberfest%20and%20status%20in%20(Open%2C%20%22To%20Do%22%2C%20Reopened)[Jenkins Jira issues suggested for Hacktoberfest]
-* link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+org%3Ajenkins-zh+is%3Aissue+is%3Aopen+label%3Ahacktoberfest[GitHub issues suggested for Hacktoberfest]
-* link:https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=19342[Newbie-friendly issue dashboard in Jenkins Jira]
-* link:https://issues.jenkins.io/issues/?jql=labels%20%3D%20newbie-friendly%20and%20status%20in%20(Open%2C%20%22To%20Do%22%2C%20Reopened)[Newbie-friendly issues in Jenkins Jira]
-* link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+org%3Ajenkins-zh+is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22[Good first issues on GitHub]
+// === Issue queries
```

Comment:
What's the purpose of this section?

Reply:
The purpose is/was to give direct links to issue queries for entry-level issues. 
The issues are currently being reviewed/triaged.

So the list/section should be published (when ready) in a second iteration.